docs/docs/deployment/platforms/on-prem.md

```diff
@@ -11,7 +11,7 @@ This guide walks you through setting up MarbleRun for your on-premises deploymen
 To deploy MarbleRun with Intel SGX, the machine or VM has to support Intel SGX.
 Particularly, MarbleRun requires support for the SGX Data Center Attestation Primitives (DCAP).
 You can verify [if your CPU supports DCAP](https://www.intel.com/content/www/us/en/support/articles/000057420/software/intel-security-products.html).
-For more information, read this article on [detecting Intel Software Guard Extensions](https://software.intel.com/content/www/us/en/develop/articles/properly-detecting-intel-software-guard-extensions-in-your-applications.html) in your applications.
+For more information, read this article on [detecting Intel Software Guard Extensions](https://www.intel.com/content/www/us/en/develop/articles/properly-detecting-intel-software-guard-extensions-in-your-applications.html) in your applications.
```

Comment:
this is also redirected once more, but to a general SGX page

Reply:
Apart from [this site](https://www.intel.com/content/www/us/en/support/articles/000058957/software/intel-security-products.html) in the Intel docs (which also contains the dead/redirecting link), I am unable to find anything pointing to the original article that was once linked here.
I would opt to simply remove this sentence, since the important things should already be explained by the article linked to in the previous sentence.